fdo.patch

```diff
@@ -0,0 +1,92 @@
+diff --git a/garglk/CMakeLists.txt b/garglk/CMakeLists.txt
```

Comment:
has this patch been submitted upstream?

Reply:
As I mentioned in the pull request, this comes from an upstream branch that will be merged after it's synchronized with any changes required for Flathub.